components/automate-cli/pkg/verifyserver/services/batchcheckservice/batchcheckservice.go

```diff
@@ -135,15 +139,21 @@ func constructBatchCheckResponse(checkTriggerRespMap map[string][]models.CheckTr
 	var result = make([]models.BatchCheckResult, len(ipMap))
 	var resultIndex = 0
 	for _, v := range ipMap {
+		if len(v) == 0 {
+			continue
+		}
 		result[resultIndex].Ip = v[0].Host
 		result[resultIndex].NodeType = v[0].NodeType
 		resultArray := []models.ApiResult{}
 		for _, checkResult := range v {
+			checkResult.Result.Message = constants.GetCheckByType(checkResult.CheckType)
```

Comment:
Why can't we make this changes to the response in one place at L47?

Reply:
@kalroy if we do it there for remote check then we need to do it for bastion check also a block up in the same file. Hence I decided to keep where the response is being constructed.

Reply:
@kalroy moved!